### PR TITLE
Decouple ButtonsPresenter from datastreams

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?
This helps decouple Argo from Fedora 3

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -60,33 +60,33 @@ class CatalogController < ApplicationController
...
+    config.add_facet_field 'rights_descriptions_ssim',          label: 'Access Rights',       component: true, limit: 1000, sort: 'index', home: false
+    config.add_facet_field 'use_license_machine_ssi',           label: 'License',             component: true, limit: 10, home: false
+    config.add_facet_field 'nonhydrus_collection_title_ssim',   label: 'Collection',          component: true, limit: 10, more_limit: 9999, sort: 'index'
+    config.add_facet_field 'hydrus_collection_title_ssim',      label: 'Hydrus Collection',   component: true, limit: 10, more_limit: 9999, sort: 'index', home: false
+    config.add_facet_field 'nonhydrus_apo_title_ssim',          label: 'Admin Policy',        component: true, limit: 10, more_limit: 9999, sort: 'index'
+    config.add_facet_field 'hydrus_apo_title_ssim',             label: 'Hydrus Admin Policy', component: true, limit: 10, more_limit: 9999, sort: 'index', home: false
+    config.add_facet_field SolrDocument::FIELD_CURRENT_VERSION, label: 'Version',             component: true, limit: 10, home: false
```
Comment (jcoyne) [inline app/controllers/catalog_controller.rb:76]: This is the only line that changed other than whitespace.

**Review by mjgiarlo — APPROVED** [review]
Do you think this ought to be tested in QA?

**Comment by jcoyne** [comment]
@mjgiarlo currently QA is unusable becauase of Mongo testing going on there. https://app.honeybadger.io/projects/50568/faults/73244757

**Comment by mjgiarlo** [comment]
@jcoyne :speech_balloon: 
> @mjgiarlo currently QA is unusable becauase of Mongo testing going on there. [app.honeybadger.io/projects/50568/faults/73244757](https://app.honeybadger.io/projects/50568/faults/73244757)

Do you think it ought to be tested in stage?

**Comment by mjgiarlo** [comment]
@jcoyne mind editing this PR to base it against main?

**Comment by jcoyne** [comment]
Andrew tested this in stage.

**Pull request merged**